content/en/agent/configuration/proxy.md

```diff
@@ -20,1209 +20,87 @@ algolia:
-
-To send traces, profiles, processes, and logs through the proxy, setup the following in the `datadog.yaml` file:
-
-```yaml
-apm_config:
-    apm_dd_url: <SCHEME>://nginx.example.com:3835
-    profiling_dd_url: <SCHEME>://nginx.example.com:3836/api/v2/profile
-    telemetry:
-        dd_url: <SCHEME>://nginx.example.com:3843
-
-process_config:
-    process_dd_url: <SCHEME>://nginx.example.com:3837
+### Apply Changes
 
-logs_config:
-    force_use_http: true
-    logs_dd_url: nginx.example.com:3838
-    # Comment the line below to use encryption between the Agent and NGINX
-    logs_no_ssl: true
+**[Restart the Datadog Agent][1]** after modifying the configuration for changes to take effect.
 
-database_monitoring:
-    metrics:
-        logs_dd_url: nginx.example.com:3839
-        # Comment the line below to use encryption between the Agent and NGINX
-        logs_no_ssl: true
-    activity:
-        logs_dd_url: nginx.example.com:3839
-        # Comment the line below to use encryption between the Agent and NGINX
-        logs_no_ssl: true
-    samples:
-        logs_dd_url: nginx.example.com:3840
-        # Comment the line below to use encryption between the Agent and NGINX
-        logs_no_ssl: true
+### Proxy Server Setup Examples
 
-network_devices:
-    metadata:
-        logs_dd_url: nginx.example.com:3841
-        # Comment the line below to use encryption between the Agent and NGINX
-        logs_no_ssl: true
-    snmp_traps:
-        forwarder:
-            logs_dd_url: nginx.example.com:3842
-            # Comment the line below to use encryption between the Agent and NGINX
-            logs_no_ssl: true
-    netflow:
-        forwarder:
-            logs_dd_url: nginx.example.com:3845
-            # Comment the line below to use encryption between the Agent and NGINX
-            logs_no_ssl: true
+If you don't have an existing proxy server, we recommend using an HTTP proxy like **Squid**.
 
-remote_configuration:
-    rc_dd_url: nginx.example.com:3846
-    # Comment the line below to use encryption between the Agent and NGINX
-    no_tls: true
-```
+1. **Squid (Recommended)**: A robust HTTP/HTTPS proxy that simplifies configuration by automatically handling Datadog endpoints. [See Squid Example Setup][2].
+2. **HAProxy (Not Recommended)**: Can forward traffic to Datadog, but this requires maintaining an up-to-date list of Datadog domains and is more complex to manage. [See HAProxy Example Setup][3].
+3. **Nginx (Not Recommended)**: Similar to HAProxy, using Nginx to forward traffic to Datadog is discouraged due to the maintenance overhead of keeping domain lists current. [See Nginx Example Setup][4].
 
+Forwarding traffic to Datadog using software like HAProxy or Nginx is **discouraged** because it requires you to manually configure and maintain the list of specific Datadog endpoints the Agent needs to reach.
```

Comment:
```suggestion
Forwarding traffic to Datadog using software like HAProxy or NGINX is **discouraged** because it requires you to manually configure and maintain the list of specific Datadog endpoints the Agent needs to reach. It should only be used if you have specific needs that cannot be addressed otherwise with a basic HTTP proxy (e.g. DPI, scaling issues, etc).
```

Reply:
This wording should probably be duplicated in each copy/pasted section.